projects/ziti-console-lib/src/lib/features/projectable-forms/configuration/configuration-form.component.ts

```diff
@@ -151,7 +151,7 @@ export class ConfigurationFormComponent extends ProjectableForm implements OnIni
     }
 
     copyCLICommand() {
-        const command = `ziti edge ${this.formData.id ? 'update' : 'create'} config ${this.formData.id ? `'${this.formData.id}'` : ''} ${this.formData.id ? '--name' : ''} '${this.formData.name}' --configTypeId '${this.formData.configTypeId}' --data '${JSON.stringify(this.formData.data)}' --tags-json '${JSON.stringify(this.formData.tags)}'`;
+        const command = `ziti edge ${this.formData.id ? 'update' : 'create'} config ${this.formData.id ? `'${this.formData.id}'` : ''} ${this.formData.id ? '--name' : ''} '${this.formData.name}' --data '${JSON.stringify(this.formData.data)}' --tags-json '${JSON.stringify(this.formData.tags)}'`;
```

Comment:
the configTypeId is positional. it shoudl be specified in case the name is changing but the `--configTypeId` should not be there

Reply:
oh nevermind, it's actually specified but under a different name :)